### PR TITLE
Fixed freeCodeCamp

### PR DESCRIPTION
**Pull request by RedS-DEV**
Changed the client id.
Presence will be maintained by me, made myself author in the metadata. (ririxi said I can do it)
Proof it works: https://reds-dev.is-inside.me/TodUZkP8.png

### CODE REVIEW

File: websites/F/FreeCodeCamp/dist/metadata.json [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
+    "id": "389383409430167562"
   },
-  "service": "FreeCodeCamp",
+  "contributors": [
+    {
+      "name": "tom",
+      "id": "266024212978728963"
+    }
+  ],
+  "service": "freeCodeCamp",
```
Comment (Bas950) [inline websites/F/FreeCodeCamp/dist/metadata.json:13]: update folder name too.

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/F/FreeCodeCamp/dist/metadata.json (hunk 1 above)
Comment (RedS-DEV) [reply websites/F/FreeCodeCamp/dist/metadata.json:13]: k

**Review by ririxidev — APPROVED** [review]

File: websites/F/freeCodeCamp/dist/metadata.json [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
+      "id": "266024212978728963"
+    }
+  ],
+  "service": "freeCodeCamp",
   "description": {
     "en": "Learn to code. Build projects. Earn certifications. Since 2015, 40,000 graduates have gotten jobs at tech companies including Google, Apple, Amazon, and Microsoft."
   },
   "url": "www.freecodecamp.org",
-  "version": "1.0.2",
+  "version": "1.1.1",
```
Comment (Bas950) [inline websites/F/freeCodeCamp/dist/metadata.json:18]: 1.1.0

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/F/freeCodeCamp/dist/metadata.json (hunk 2 above)
Comment (RedS-DEV) [reply websites/F/freeCodeCamp/dist/metadata.json:18]: k

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**